ert/data/record/_record.py

```diff
@@ -144,7 +144,8 @@ def __init__(
 
         Args:
             data: numerics that cannot be None
-            index: data indices, which defaults to None.
+            index: data indices. If None is given, then index will be
+                inferred from the data.
```

Comment:
-> "the index will be an 0-indexed enumeration of the elements in the data."

Reply:
Nice, just thought that this can change in the future so wanted to keep it general. But we should always do as it is presently - so updating it then :)